### DISCUSSION TITLE
Use of color to confirm the pre-requisites are met?

### DISCUSSION THREAD

**Discussion by plearn2023** [vmware/powershell-module-for-vmware-cloud-foundation-reporting#114] [Community Help]
Can we add a green color, it is easier to confirm that all pre-requisites are passed?

```powershell
PS C:\Users\administrator.CORP\Downloads> Test-VcfReportingPrereq
PowerShell Module: VMware.PowerCLI Version: 13.0.0.20829139 Found, Supports the minimum required version.
PowerShell Module: VMware.vSphere.SsoAdmin Version: 1.3.9 Found, Supports the minimum required version.
PowerShell Module: PowerVCF Version: 2.2.0 Found, Supports the minimum required version.
PowerShell Module: PowerValidatedSolutions Version: 2.1.0.1002 Found, Supports the minimum required version.
```

**Comment by tenthirtyam** [comment]
That's a nice idea. I'll add this to the module.

**Reply by tenthirtyam** [reply]
~~Tracked under https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/issues/115.~~

**Reply by tenthirtyam** [reply]
Thinking this over a little bit, I've closed the above. We use `Write-Output`, `Write-Warning`, and `Write-Error` which control the color. I may modify the message for clarity.

**Reply by tenthirtyam** [reply]
An incoming pull request that will use our `Write-LogMessage` function from the `PowerVCF` module dependency.

<img width="1205" alt="image" src="https://user-images.githubusercontent.com/7771363/231239632-ca9d9f62-475d-4c55-8f69-a0e35f07bde2.png">

<img width="1498" alt="image" src="https://user-images.githubusercontent.com/7771363/231260545-0ca5495e-f8c5-456a-8345-8a6cd78d1b0e.png">

**Reply by tenthirtyam** [reply]
Addressed in pull request https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/117.
